lib/src/rules.dart

```diff
@@ -22,6 +22,8 @@ import 'package:linter/src/rules/avoid_function_literals_in_foreach_calls.dart';
 import 'package:linter/src/rules/avoid_implementing_value_types.dart';
 import 'package:linter/src/rules/avoid_init_to_null.dart';
 import 'package:linter/src/rules/avoid_js_rounded_ints.dart';
+import 'package:linter/src/rules/avoid_mutable_fields_in_equality_operators.dart';
```

Comment:
This one is missing from this PR I think?

Reply:
Whoops, that is left over from a different approach. I need to remove it.

Reply:
Done